refs/etsi.json

```diff
@@ -11762,7 +11762,7 @@
     },
     "etsi-en-302-120": {
         "href": "http://www.etsi.org/deliver/etsi_en/302100_302199/302120/01.01.01_60/en_302120v010101p.pdf",
-        "title": "ETSI EN 302 120 V1.1.1 (2003-01): Services and Protocols for Advanced Networks (SPAN); Operation of the Bearer Independant Call Control (BICC) protocol with SIG.4/PNNI1.0/AINI signalling specification [ATM Forum specification AF-CS-VMOA-0146.000 (2000) modified]",
+        "title": "ETSI EN 302 120 V1.1.1 (2003-01): Services and Protocols for Advanced Networks (SPAN); Operation of the Bearer Independant [sic] Call Control (BICC) protocol with SIG.4/PNNI1.0/AINI signalling specification [ATM Forum specification AF-CS-VMOA-0146.000 (2000) modified]",
```

Comment:
I've generally chosen to tag instances where the underlying content has a typo with `[sic]`.

Reply:
I'm not sure about this, tbh?

Maybe just leave those as is?

Reply:
Yeah, I'm happy to drop them. I did it mostly so I could have a temporary record of which things I had checked and confirmed were bugs upstream as opposed to incorrectly imported.